community/contributing/index.md

```diff
@@ -22,3 +22,8 @@ In the meantime, if you're really excited about contributing and don't want to w
 
 For more details about the patch submission process, look at [How to submit a patch](/community/contributing/submitting/).
 
+## How to run Spinnaker Locally
+
+The [environments](/setup/install/environment/#local-git) page offers a guide for using Halyard to configure a local git repository consistent with a released version of Spinnaker. This can be a great option for developers looking to contribute to Spinnaker to make and test their changes before sending any code for review.
```

Comment:
Consider deleting the 2nd sentence of this paragraph, and adding a single-sentence paragraph before this one. Something like, “Running Spinnaker locally on your machine is a great option for testing changes before submitting a pulll request.”
I suggest this as a way of letting the reader know immediately why these instructions are here on the Contributing page.

Reply:
Done.